modular_citadel/code/modules/custom_loadout/custom_items.dm

```diff
@@ -439,3 +436,23 @@
 	icon_state = "moveralls"
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
 
+/obj/item/clothing/head/telegramhat
+	name = "Singing Telegram Hat"
+	icon = 'modular_citadel/icons/obj/clothing/cit_hats.dmi'
```

Comment:
It's still missing the alternate_worn_icon, That's located in `modular_citadel/icons/obj/clothing/turtlenecks.dmi` for some reason.